Display.cpp

```diff
@@ -2,6 +2,10 @@
 #include "global.h"
 #include "time_util.h"
 #include "localization.h"
+#ifdef BIRTHDAYS
+#include "birthday.h"
+Birthday* activeBirthday = nullptr;
```

Comment:
This doesn't need to be a global

Reply:
I put this at this level so it would not import into the project unless defined in config to keep memory lower when not used.  Where would you propose this import be moved?  I can't move birthday.h into config so not sure where else but global this could be.

Reply:
I'm just talking about the declaration of activeBirthday, not the import. That variable is only used inside the update function so it can be local to that.